### ISSUE TITLE
Package not installable via pip on Python 3.4.2 / virtualenv / Debian 8

### ISSUE THREAD

**Issue by tomwll** [fmnisme/python-icinga2api#4]
The pip package is not installable on a Debian Jessie system and the following environment:

Python version: 3.4.2
Debian Packages: python3, python3-virtualenv
virtualenv --version: 1.11.6

How to reproduce:
```
apt-get install python3 python3-virtualenv
virtualenv -p python3 v1
source v1/bin/activiate
pip install python-icinga2api
```

Expected result:
Successful installation

Actual result:
Python error message:
```
Collecting python-icinga2api
  Using cached python-icinga2api-0.3.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-amltcudf/python-icinga2api/setup.py", line 25, in <module>
        long_description=read('README.md'),
      File "/tmp/pip-build-amltcudf/python-icinga2api/setup.py", line 13, in read
        return open(os.path.join(os.path.dirname(__file__), fname)).read()
    FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-build-amltcudf/python-icinga2api/README.md'
```

**Comment by fmnisme** [comment]
It just support python2.x

**Comment by danielparks** [comment]
I'm running into the same error with Python 2.7 on OS X 10.11.6:

~~~
(pi2c) ❯ uname -a
Darwin marla.local 15.6.0 Darwin Kernel Version 15.6.0: Mon Jan  9 23:07:29 PST 2017; root:xnu-3248.60.11.2.1~1/RELEASE_X86_64 x86_64
(pi2c) ❯ python --version
Python 2.7.13
(pi2c) ❯ pip --version
pip 9.0.1 from /Users/daniel/.virtualenv/pi2c/lib/python2.7/site-packages (python 2.7)
(pi2c) ❯ pip --no-cache-dir install python-icinga2api
Collecting python-icinga2api
  Downloading python-icinga2api-0.3.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/snip/T/pip-build-08yJgC/python-icinga2api/setup.py", line 25, in <module>
        long_description=read('README.md'),
      File "/private/var/folders/snip/T/pip-build-08yJgC/python-icinga2api/setup.py", line 13, in read
        return open(os.path.join(os.path.dirname(__file__), fname)).read()
    IOError: [Errno 2] No such file or directory: '/private/var/folders/snip/T/pip-build-08yJgC/python-icinga2api/README.md'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/snip/T/pip-build-08yJgC/python-icinga2api/
~~~

**Comment by danielparks** [comment]
PR #5 fixes the installation in Python 2.7 and 3.6. I haven't tested using the package in Python 3.6.

**Comment by davewongillies** [comment]
Can you publish 0.3.1 to pypi? 0.3 is still the current version there.
